src/client/cli/cmd/networks.h

```diff
@@ -15,8 +15,8 @@
  *
  */
 
-#ifndef MULTIPASS_LIST_COMMANDS_H
-#define MULTIPASS_LIST_COMMANDS_H
+#ifndef MULTIPASS_NETWORKS_H
+#define MULTIPASS_NETWORKS_H
```

Comment:
Oops, my fault. I forgot these guards when renaming the commands. Thanks @ricab for fixing this :)